src/content/developers/docs/layer-2-scaling/index.md

```diff
@@ -18,6 +18,53 @@ You should have a good understanding of all the foundational topics. Implementin
 - Any updates to scalability should not be at the expense of decentralization of security – layer 2 builds on top of Ethereum.
 
+## Rollups {#rollups}
+
+Rollups are solutions that bundle or "roll up" sidechain transactions into a single transaction and generate a cryptographic proof, known as a SNARK (succinct non-interactive argument of knowledge). Only this proof is submitted to the main chain.
+
+_Sidechains are Ethereum-compatible, independent blockchains._
+
+In other words, rollups mean that all state and execution is handled in sidechains – signature verification, contract execution, etc. The main Ethereum chain only stores transaction data.
+
+Rollup solutions require relayers who have staked a bond in the rollup contract. This incentivises them to relay rollups accurately.
+
+**Useful for:**
+
+- reducing fees for users
+- open participation
+- fast transaction throughput
+
+There are two types of rollups:
+
+- Zero knowledge – can only handle simple transactions
+- Optimistic – can run anything Ethereum can using an optimistic version of the EVM, the Optimistic virtual machine
+
+### Optimistic rollups
+
+Optimistic rollups use a side chain that sits in parallel to the main Ethereum chain. They can offer improvements in scalability because they don't do any computation by default. Instead, after a transaction they propose the new state to mainnet. Or "notarise" the transaction.
+
+As computation is the slow, expensive part of using Ethereum, this can offer up to 10-100x improvements in scalability dependent on the transaction. This number will increase even more with the introduction of the Eth2 upgrade: [shard chains](/en/eth2/#shard-chains). This is because there will be more data available in the event that a transaction is disputed.
+
+#### Disputing transactions
+
+Optimistic rollups don't actually compute the transaction, so there needs to be a mechanism in place to ensure transactions are legitimate and not fraudulent. This is where fraud proofs come in. If someone notices a fraudulent transaction, the rollup will execute a fraud-proof and run the transaction's computation, using the available state data.
+
+![Diagram showing what happens when a fraudulent transaction occurs in an Optimistic rollup in Ethereum](./optimistic-rollups.png)
+
+The gas you need to run the computation of the fraud proof is even reimbursed. Ben Jones from Optimism describes the bonding system in place:
+
+"_anyone who might be able to take an action that you would have to prove fraudulent to secure your funds requires that you post a bond. You basically take some ETH and lock it up and you say "Hey, I promise to tell the truth"... If I don't tell the truth and fraud is proven, this money will be slashed. Not only does some of this money get slashed but some of it will pay for the gas that people spent doing the fraud proof_"
+
+So you get reimbursed for proving fraud.
+
+#### The Optimism virtual machine (OVM)
```

Comment:
I think this is a decent overview of the OVM but for us to go into this, I think there needs to be an explanation that Optimism (& the OVM approach) is not the only implementation of optimistic rollups. e.g. Fuel Labs has an implementation of optimistic rollups that uses a [UTXO based transaction system](https://docs.fuel.sh/v1.0.0/Concepts/Fundamentals/Fuel%20Overview.html#architectureahighlevelview) so as a result, it doesn't have the OVM or EVM-like execution environment. It optimizes for a limited use case:  [ERC-20 transfers](https://docs.fuel.sh/v1.0.0/Introduction/Use%20Cases.html).

In short, I think we need to explain that within rollups, there are 2 approaches in terms of security model:
- ZK rollups (validity proofs)
- Optimistic rollups (fraud proofs)

Even within these 2 categories, there are still a range possible implementations, with multiple projects taking different approaches that also have their own set of tradeoffs.

Reply:
Ah, I didn't realise OVM was just one brand's implementation of Optimistic rollups. Perhaps we should remove this section rather than us have to commit to explaining every brand's implementation. We could replace it with links to OVM and UTXO etc...

Reply:
Ya I'm fine with just removing the section on OVM. Perhaps in the general rollups section just explain that there's different implementations that optimize for different sets of tradeoffs.